index.js

```diff
@@ -0,0 +1 @@
+alert("Hello World");
```

Comment:
Do we need this?

Reply:
Both done, npm package only includes js files and ignores others.